feature-libs/checkout/components/components/checkout-progress/checkout-progress.component.html

```diff
@@ -1,4 +1,7 @@
-<section *ngIf="(activeStepIndex$ | async) !== undefined">
+<nav
```

Comment:
Should be transferred

Reply:
don't worry! as explained here https://sap-cx.slack.com/archives/D02A0NHD3C0/p1643298314004500?thread_ts=1643268177.004900&cid=D02A0NHD3C0